src/main/webapp/WEB-INF/views/admin/partner/edit-js.jsp

```diff
@@ -2022,5 +2022,32 @@ var sampleFile =  ${ct:getAllValuesJson(ct:getConstDef('CD_SAMPLE_FILE'))};
 			}
 		}
 	}
+
+	var src_fn = {
+        // License 상세 페이지 이동
+        showLicenseViewPage : function(gridNm, rowid){
+            console.log("test : " + gridNm);
```

Comment:
It would be a good idea to remove console.log when commiting

Reply:
Thanks for your comments.
Pull Request still in progress, so I'll reflect it when it's complete!

Reply:
@astrod 
I've reflected your opinion Thank you for commenting!